features/HistoricalRequest.feature.md

```diff
@@ -49,7 +56,7 @@ And I send this message via the websocket
   "message": "message",
   "payload": {
     "@context": "https://github.com/hello-nrfcloud/proto/historical-data-request",
-    "@id": "420eac59-5ce8-4751-b7d1-217811382095",
+    "@id": "${requestId}",
```

Comment:
Unrelated change. What's wrong with using a hardcoded ID here?

Reply:
When I debug the freaky test, I would like to have a unique request id. So, it is easier to search on the log.